analyses/cell-type-wilms-tumor-06/notebook-template/01-clustering.Rmd

```diff
@@ -0,0 +1,768 @@
+---
+title: "Evaluation of clustering parameters"
+author: maud-p
+date: '`r format(Sys.time(), "%B %d, %Y %H:%M:%S %Z")`'
+output:
+  html_document: 
+    toc: yes
+    toc_float: yes
+    code_folding: hide
+    highlight: pygments
+    df_print: kable
+params:
+  sample_name: ~
+  sample_path: ~
+  out_rds_path: ~
```

Comment:
As written in my overall comment, you'll want to use some actual defaults there that will allow the notebook to run.

Reply:
thanks, I didn't know about the params, I'll try to use it starting from now!

Reply:
They're super useful! Here's some info about them 🙂:
- https://docs.posit.co/connect/user/param-rmarkdown/
- https://rmarkdown.rstudio.com/lesson-6.html

Reply:
Thank you!